.github/workflows/accuracy_test.yaml

```diff
@@ -258,7 +258,7 @@ jobs:
           TIMESTAMP=$(date +%Y%m%d%H%M%S)
           BRANCH_NAME="auto-pr/accuracy-report-${TIMESTAMP}"
           echo "BRANCH_NAME=${BRANCH_NAME}" >> $GITHUB_ENV
-          git checkout -B "${BRANCH_NAME}" upstream/${{ github.event.inputs.vllm-ascend-version }}
+          git checkout -B "${BRANCH_NAME}" upstream/main
```

Comment:
That means currently we only support submit report in main branch:

<img width="471" height="312" alt="image" src="https://github.com/user-attachments/assets/2f35360b-b142-4e6e-b0b4-8093a43f7293" />

We should think how to make it more general for all branches, first select the branch, and use the branch to run vllm-ascend branch.